test/over_react/util/dom_util_test.dart

```diff
@@ -154,31 +154,41 @@ main() {
     });
   });
 
-  group('setSelectionRange', () {
-    test('throws an ArgumentError if called on an unsupported Element type', () {
+  group('setSelectionRange and getSelectionStart: ', () {
```

Comment:
I think these tests are better suited to separate groups - although they're related methods, the test descriptions aren't as clear as they could be and separated tests would be more atomic/unit-ish. So something that looks like:

```dart
group('setSelectionRange', () {
  test('throws an ArgumentError if called on an unsupported Element type', () {
    ...
  });
  ...
});

group('getSelectionStart', () {
  test('returns normally if called on an unsupported Element type', () {
    ...
  });
  ...
});
```